_posts/0000-01-02-CaptKanami6.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: “Welcome to our second slide!”
+---
+Your test
+Use the left arrow to go back!
```

Comment:
n

Reply:
---
layout: slide
title: “Welcome to our second slide!”
---
Your test
Use the left arrow to go back!